x-pack/platform/packages/shared/index-management/index_management_shared_types/src/types.ts

```diff
@@ -35,9 +36,15 @@ export type IndexManagementLocatorParams = SerializableRecord &
 
 export type IndexManagementLocator = LocatorPublic<IndexManagementLocatorParams>;
 
+export type IndexManagementAppMountParams = Pick<
+  ManagementAppMountParams,
+  'setBreadcrumbs' | 'history'
+> & { element: HTMLDivElement | HTMLElement | null };
```

Comment:
Similar comment here I think this can be simplified to 
```suggestion
> & { element: HTMLElement | null };
```

Reply:
yes, you are correct. My bad. We don't need `HTMLDivElement` but need to explicitly have a new type for `element`, since we need element to accept `null` type as well,  which [ManagementAppMountParams](https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/management/public/types.ts#L78) doesn't 

updated in https://github.com/elastic/kibana/pull/209599/commits/3343d3721544521ec0f49a2486e904e29903dd04